src/checkmatelib/exceptions.py

```diff
@@ -35,7 +35,7 @@ def handles_request_errors(inner):
     """Translate requests errors into our application errors."""
 
     @wraps(inner)
-    def deco(*args, **kwargs):
+    def deco(*args, **kwargs):  # pylint: disable=inconsistent-return-statements
```

Comment:
pylint version it's not pinned, newer version raised this error on CI and locally with manual upgrade to pylint=2.7.1 even if nothing changed here on this PR